packages/cinterop/src/nativeDarwin/kotlin/io/realm/kotlin/internal/interop/RealmInterop.kt

```diff
@@ -3015,6 +2803,55 @@ actual object RealmInterop {
     }
 }
 
+fun realm_value_t.asByteArray(): ByteArray {
```

Comment:
Huh, I can't place them inside the `RealmValue` implementations because of an apparent conflict with the inline class 🤔 Even if I put these outside the class in the same file I get an error that the static function cannot be found 🤯